src/symbolize.cc

```diff
@@ -780,8 +781,13 @@ static ATTRIBUTE_NOINLINE bool SymbolizeAndDemangle(void *pc, char *out,
                                                              out_size - 1);
   }
 
+#if defined(PRINT_UNSYMBOLIZED_STACK_TRACES)
```

Comment:
could we add this for autoconf and/or cmake?

Reply:
Sure. Updated.